lib/uffizzi/cli/preview.rb

```diff
@@ -233,6 +238,14 @@ def handle_preview_interruption(deployment_id, server, project_slug)
       raise Uffizzi::Error.new("The preview creation was interrupted. #{preview_deletion_message}")
     end
 
+    def handle_preview_error(deployment_id, error)
+      if Uffizzi.ui.output_format.nil?
+        raise error
+      else
+        raise Uffizzi::Error.new("Preview with ID deployment-#{deployment_id} failed.")
+      end
+    end
+
```

Comment:
I think we should not use `failed` error for every `Uffizzi::Error`. It would be more properly if we change error message `Deployment failed` on backend (uffizzi_app repository). 
Let's change message here:

```
# core/config/locales/en.yml

  deployment:
    # now
    invalid_state: Deployment %{state}
    
    # should be
    invalid_state: Preview with ID deployment-%{deployment_id} %{state}
```